api/routers/objects.py

```diff
@@ -25,16 +24,19 @@ def get_owner_objects(bucket, owner, prefix=None, limit=1000, skip=0):
 
 @router.put("/{bucket}/{key:path}")
 async def create_object(
-    bucket: str,
-    key: str,
-    request: Request,
-    owner: str = Depends(extract_token),
+        bucket: str,
```

Comment:
I see these formatting changes
Can you please install pre-commit hook and commit these files again?